src/api/index.js

```diff
@@ -8,7 +8,7 @@ const API = axios.create({
   baseURL: 'https://api.github.com',
   headers: {
     'Content-Type': 'application/json',
-    Authorization: `token ${process.env.REACT_APP_API_KEY}`
+    Authorization: `token ${'process.env.REACT_APP_API_KEY'}`
```

Comment:
We can't change this line because after putting **process.env.REACT_APP_API_KEY** in **single quotes**, we won't be able to load the token from the .env file. it will break our app.
To run this app follow these steps:-

1. Create a .env file,
2.  copy contained from from.env.sample file and paste it into the .env file,
3. create your own key generated from here https://github.com/settings/tokens
4. put it in the .env file, in place of **gh_xxxxxxxx...** i.e, after **REACT_APP_API_KEY=**

Now you are good to go. 👍